client/js/components/GameView/GameDialogs.vue

```diff
@@ -121,7 +122,8 @@ export default {
         this.resolvingSeven &&
```

Comment:
We can use `playingFromDeck` instead of `resolvingSeven` because `playingFromDeck` gets set when you're resolving a seven _and_ it's your turn
```suggestion
        this.playingFromDeck &&
```

Reply:
This change does the trick for the opponent view, but causes tests for playing a seven into a double jack scenario to fail.

Reply:
That surprises me, huh. I wonder why

Reply:
^^ Per our conversation, I added `playingFromDeck: ({ game }) => game.playingFromDeck` to a new mapState expression in GameDialogs. Works as intended